packages/expo/cli/utils/analytics/rudderstackClient.ts

```diff
@@ -0,0 +1,99 @@
+import RudderAnalytics from '@expo/rudder-sdk-node';
+import os from 'os';
+import { v4 as uuidv4 } from 'uuid';
+
+import { EXPO_LOCAL, EXPO_STAGING, EXPO_NO_TELEMETRY } from '../env';
+import UserSettings from '../user/UserSettings';
+
+const PLATFORM_TO_ANALYTICS_PLATFORM: { [platform: string]: string } = {
+  darwin: 'Mac',
+  win32: 'Windows',
+  linux: 'Linux',
+};
+
+let client: RudderAnalytics | null = null;
+let identified = false;
+let identifyData: {
+  userId: string;
+  deviceId: string;
+  traits: Record<string, any>;
+} | null = null;
+
```

Comment:
should wrap all these methods in an Analytics class like in XDL https://github.com/expo/expo-cli/blob/main/packages/xdl/src/Analytics.ts

e.g. create an internal analytics class here, then export two versions of it with different write keys and have the caller specify which client they need

```
class AnalyticsClient {
  private userTraits?: { [key: string]: any };
  private rudderstackClient?: RudderAnalytics;
  private _userId?: string;
  private _version?: string;
  ....
}

const xdlClient = new AnalyticsClient();
const unifiedClient = new AnalyticsClient();

export { xdlClient, unifiedClient };
```